controller/src/beerocks/cli/beerocks_cli_main.cpp

```diff
@@ -279,6 +277,8 @@ static void cli_interactive(std::string path, std::string tmp_path, std::string
         is_onboarding = cli_bml.get_onboarding_status();
     }
 
+    beerocks::cli_socket cli_soc(tmp_path, ip);
```

Comment:
What? Why?

Reply:
I wrote why in the commit message.
I guess the `cli_bml.connect()` is taking too long so it prints the warning message I added to the socket_thread.